Cargo.toml

```
@@ -20,6 +20,12 @@ event-listener-strategy = { version = "0.5.2", default-features = false }
 futures-core = { version = "0.3.5", default-features = false }
 pin-project-lite = "0.2.11"
 
+# This dependency is unused, but it lets us add features to it.
+__event-listener = { version = "5", default-features = false, package = "event-listener" }
+
+portable-atomic = { version = "1", default-features = false, features = ["require-cas", "fallback", "critical-section"], optional = true }
```

```suggestion
portable-atomic = { version = "1", default-features = false, features = ["require-cas"], optional = true }
```
`fallback` and `critical-section` should be enabled by the consumers of this crate, not this library itself, since they can conflict with other features (unsafe assume single core for example). For testing in CI, you'd be able to call:
```sh
cargo check --no-default-features --target thumbv6m-none-eabi --features portable-atomic,portable-atomic/fallback,portable-atomic/critical-section 
```

---

makes sense. Would we need to add that to bevy CI though?